lineman/app/components/thread_page/proposal_pie_chart/proposal_pie_chart.scss

```diff
@@ -50,8 +38,8 @@
   color: $grey-on-white;
 }
 
-@media (min-width: $medium-max-width) and (max-width: $large-max-width) {
-  .proposal-pie-chart__legend{
-    display: block;
+@media (max-width: $medium-max-width) {
+  .proposal-pie-chart__legend {
```

Comment:
This doesn't work for me between 760px and 840px width, and should be floating right from the pie chart from 480px - 760px. Not sure why we need to change this, actually.

Might consider trying `display: flex` out here.